messenger/make.py

```diff
@@ -67,9 +76,55 @@ def build_octave():
     do_build(make_cmd, 'messenger.mex')
 
 
+def which_matlab():
```

Comment:
Also dude, it wasn't bad to read; it's just a bit different than my coding
style so.

On Fri, Apr 10, 2015 at 5:05 PM, Ali Ebrahim <aebrahim@ucsd.edu> wrote:

> Honestly, it was easier for me to rewrite this than try to debug it when
> it didn't work on Windows when I tried to merge your PR.
>
> Is it possible to just apply the simplifications you need (like 1 config
> file, integration with setup.py) using the functions in my PR?
>
> On Fri, Apr 10, 2015 at 5:00 PM, Sang Han <notifications@github.com>
> wrote:
>
>> In messenger/make.py
>> <https://github.com/arokem/python-matlab-bridge/pull/178#discussion_r28189008>
>> :
>>
>> > @@ -67,9 +76,55 @@ def build_octave():
>> >      do_build(make_cmd, 'messenger.mex')
>> >
>> >
>> > +def which_matlab():
>>
>> So the structure can be different.
>>
>> I know that sometimes there is a R2014_* directory that is present in
>> windows but not on OSX or Linux. I took care of that with this code.
>>
>> Sorry, maybe I should have written more comments. Honestly didn't even
>> think anyone would even read this :)
>>
>>         for p in os.listdir(os.path.join(programs[host], *matlab)):
>>             if p.startswith('R20'):
>>                 matlab.append(p)
>>
>>         matlab.append('bin')
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/arokem/python-matlab-bridge/pull/178/files#r28189008>
>> .
>>
>
>